packages/cloud-toolkit/src/services/update-typings.ts

```diff
@@ -21,5 +25,30 @@ export default async function updateTypings(
       introspection: await httpForestServer.getIntrospection(),
     }),
   );
+
```

Comment:
```suggestion
export default async function loadCustomization(agent: Agent, path: string) {
  if (!fs.existsSync(path)) {
    logger('Info', `Customization file not found at ${path}`);

    return;
  }

  const customization = require(path);

  const entryPoint = customization?.default || customization;

  if (typeof entryPoint !== 'function') {
    throw new Error('Customization file must export a function');
  }

  await entryPoint(agent);
}
```
https://github.com/ForestAdmin/cloud-agent-manager/blob/0963fe110efdfbf4d9835d5652e554d59295cb23/packages/lambda-layers/src/agent/internal/load-customization.ts#L9 

This is how we handle it in the lambda, the code should be pretty similar.